exercises/concept/last-will/.docs/instructions.md

```diff
@@ -0,0 +1,33 @@
+# Instructions
+
+You work for a prestigious law firm that is specialized in handling unique testament requests.
+
+In this exercise, you are going to open a mysterious vault.
+You are the executor of the estate and will assemble the long-kept secret codes of three families to get an account number and the matching code.
+
+To prevent any family from opening the vault alone, it can only be opened by combining their knowledge with a secret modifier that you know from the last will.
+
+You have three tasks, all related to helping the families to open the vault.
+
+## 1. Define a Namespace for the executor of estate functions
```

Comment:
Tiny nit: these task headings are very technical. In general, we try to make these more like story-based tasks. E.g. https://github.com/exercism/fsharp/blob/main/exercises/concept/bandwagoner/.docs/instructions.md#2-create-a-teams-coach lists the task as "Create a team's coach", and not as "Create a coach record". But if this rephrasing is hard for this exercise, it's totally fine.

Reply:
I think I made it more story-like now.